src/components/profile/Recovery/RecoveryContainer.js

```diff
@@ -91,7 +92,7 @@ const RecoveryContainer = () => {
         missingKinds.forEach(kind => activeMethods.push({kind: kind}));
     }
 
-    const loading = account.actionsPending.includes('LOAD_RECOVERY_METHODS') || account.actionsPending.includes('REFRESH_ACCOUNT');
+    const loading = actionsPending(['LOAD_RECOVERY_METHODS', 'REFRESH_ACCOUNT'])
```

Comment:
I think generally makes sense to refactor this in similar way to alerts – have flag in `meta` whether action should be part of `loading`. By default makes sense to count everything in global loading status.

Reply:
I'm not sure if it would be handy - the problem is that we might want to use some action as a loader in one place and not use it in another. I will take a closer look and check if a situation like this appears.